chromium_src/components/signin/internal/identity_manager/primary_account_manager.cc

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "components/signin/internal/identity_manager/primary_account_manager.h"
+
+#include "extensions/common/url_pattern.h"
+#include "google_apis/gaia/gaia_urls.h"
+#include "url/gurl.h"
+
+// Ensures that requests to accounts.google.com is not initiated at startup if
+// google login for extensions is enabled
+#define BRAVE_NO_GAIA_REQUEST_ON_STARTUP                  \
```

Comment:
Pretty sure that these defines should be named based on the class/function they're in, instead of by their use, but @bridiver can verify.